CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## 1.8.2
+* Enhancement - Added a functional test for a now fixed TaskEngine deadlock issue.
```

Comment:
Sounds good. I'll update the changelog to:

Bug - Fixed an issue where exec_create and exec_start events were not correctly ignored with some Docker versions.
Bug - Resolved a bug where sending a signal to a container caused the agent to treat the container as dead.